packages/tasit-sdk/package.json

```diff
@@ -18,9 +18,12 @@
   },
   "scripts": {
     "prepare": "npx babel src --out-dir dist --ignore **/*.test.js",
-    "test": "npm run test:local",
-    "test:local": "npm run lint && npx mocha src/*.test.js --exclude src/*forkedRopsten.test.js --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
-    "test:forkedRopsten": "npm run lint && npx mocha src/*forkedRopsten.test.js --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
+    "test": "npm run lint && npm run test:local",
+    "test:local": "cp src/config/development.js src/config/default.js && npm run test:run",
+    "test:forked-testnet": "npm run test:forked-ropsten",
+    "test:forked-ropsten": "cp src/config/forked-ropsten.js src/config/default.js && npm run test:run",
+    "test:forked-goerli": "cp src/config/forked-goerli.js src/config/default.js && npm run test:run",
```

Comment:
Remind me in what sense it is a forked testnet? Do we copy its state locally? Or does it just stop receiving updates to the real testnet but run remotely?

Reply:
`ganache-cli` has the `--fork <NODE_URL>` option. 
I don't know how exactly it works in details but seems that `ganache` start a local blockchain from the last testnet state.

> --fork feature meant to solve the issues with dynamic analysis on the live chain (or any chain, for that matter). It did so by allowing you to "fork" from the live chain -- i.e., create a new blockchain that starts from the last block on the existing chain -- allowing you to make transactions on the new chain without spending real Ether or changing the existing chain's state.

Refs: https://truffleframework.com/tutorials/chain-forking-exploiting-the-dao#hello-testrpc

Reply:
Ah ok yes, makes sense. Yep, it does seem that it starts a local blockchain. Cool. Resolving this.